server/src/main/java/com/google/musicanalysis/site/YoutubeServlet.java

```diff
@@ -106,6 +110,16 @@ protected void updateMusicCount(String youtubeResBody, HashMap<String, Integer>
         return;
     }
 
+    /**
+     * @param youtubeJsonObj json obj of youtube response body 
+     * @return {int} number of total results from json response
```

Comment:
nit: should not have types in JSDoc comments in Java, since Java has types on everything